bin/crew

```diff
@@ -467,6 +470,7 @@ def help(pkgName = nil)
     puts <<~EOT
       Remove and install package(s).
       Usage: crew reinstall [-k|--keep] [-s|--source] [-S|--recursive-build] [-v|--verbose] <package1> [<package2> ...]
+      If `-f` or `--force` is present, an option is forced, such as in `crew build -f packagename`.
```

Comment:
Do we need this here?

Reply:
It's sort of obtuse otherwise, but I can remove it ...

Reply:
Done.